rfcs/0013-synthea-future-claims-and-automation.md

```diff
@@ -0,0 +1,130 @@
+- The RIF files are copied into the BFD Server codebase, and loaded into the local database. 
+- If no errors loading locally, the RIF files are uploaded to AWS S3 and loaded into TEST, PROD SBX, PROD databses via the same ETL process with CCW RIF files. 
+- Synthetic data is now available to BFD users.
+
+* In addition to these steps, BFD follows a thorough test plan before generating or loading data into TEST, PROD SBX, or PROD environments. 
+  * [Synthea Test Plan](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md) 
+
+Motivation:
+With previous Synthea data releases having spanned about 3 months apart, and only 20,000 synthetic beneficiaries released, data becomes easily outdated, and the amount of data being released doesn't justify the usecase of synthetic data. So, there is inefficiency involved with producing and releasing synthetic data when done without automation. Inefficiency that in part is caused by a process full of human erorr, causing incidents, such as accidental overlap of beneficiary IDs between different synthetic data batches. Therefore, there are more than enough reasons to look into streamlining and scaling up synthetic data production to maximize the benefit to users and partners. 
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+The proposed solution is to automate the generation of synthetic data with a set batch size on a weekly basis using the latest version of the synthea codebase.
+Each quarter, the expectation is to have significant changes to the synthetic data properties from Mitre's code. So the communications to users and partners on a quarterly basis will be more in depth compared to that of on a weekly basis. The goal is to primarly generate data on a regular basis to increase the benefit and quality of the data to the user at a given point in time. 
+
+On a weekly basis, synthetic data with a future claim date will also be generated. This data will be staged until it can be loaded properly with the claim date. 
+
+Additionally, for this data to be ingested on a weekly basis, the BFD ingestion pipeline will need to work with Synthea data as it does with CCW data, for which back-dated beneficiary filtering will be turned off for Synthea data. 
+
+When producing and loading a large set of synthetic data and performance testing:
+    - Size and shape of the data will reflect production
+    - Will be available for on-demand performance and load testing
+    - Will not be released. Specifically, not released to PROD.
+    - Will be isolated from other released synthetic data. 
+    - Will have the capability to be re-generated with future versions of synthea
+    - Synthetic beneficiaries for load and performance testing will not overlap with any beneficiaries used by end users
```

Comment:
I think this is just the requirements from the story and some of them we were backtracking on in the brainstorming meeting. These are just things to consider as you are describing how we generate a large data set, what it is for, where it lives, etc.

Reply:
We should also comment here about the fact that we are getting rid of the old synthetic data and try to be somewhat specific about what that means (which data will be removed, does that free up ranges that make it easier for us to populate recurring and/or large data sets? (it should))

Also, what sort of ranges of identifiers are we going to use for the large data? Will it be segregated from the recurring. I think it probably should be and we can propose something pretty specific for that.

(BTW, we should probably have a small bit of education about synthetic data in the background. things like -- synthetic bene_ids are negative and they have an S character in a certain position of the MBI, etc)